reverse-proxy/nginx/nginx.conf

```diff
@@ -42,13 +42,13 @@ http {
             # CORS...
             add_header 'Access-Control-Allow-Origin' '*' always;
             add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS' always;
-            add_header 'Access-Control-Allow-Headers' 'Authorization,DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range' always;
+            add_header 'Access-Control-Allow-Headers' 'Authorization,DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,company' always;
```

Comment:
does this need to be `Company` vs. `company` ?

Reply:
i think these get lower cased automatically anyway. But all our references to it are 100% lower case